ApplicationLibCode/ProjectDataModel/CellFilters/RimCellFilterCollection.cpp

```diff
@@ -316,6 +317,8 @@ void RimCellFilterCollection::setAutoName( RimCellFilter* pFilter )
 void RimCellFilterCollection::onChildDeleted( caf::PdmChildArrayFieldHandle* childArray, std::vector<caf::PdmObjectHandle*>& referringObjects )
 {
     onFilterUpdated( nullptr );
+
+    filtersChanged.send();
 }
```

Comment:
Is this send necessary? onFilterUpdated calls send, too.

Reply:
That's true. Good point!